src/app/(pages)/codeLibrary/page.tsx

```diff
@@ -0,0 +1,491 @@
+
+    return conditionDetails ? formatDiseaseDisplay(conditionDetails) : "";
+  };
+
+  const formatValueSetDetails = (vs: DibbsValueSet) => {
+    const system = formatSystem(vs.system) || "";
+
+    const conceptType = vs.dibbsConceptType
+      ? `${formatStringToSentenceCase(vs.dibbsConceptType)} ${
+          !!system ? " • " : ""
+        }`
+      : "";
+
+    const condition = vs.conditionId
+      ? `${formatConditionDisplay(vs.conditionId)} ${
+          !!conceptType ? " • " : ""
+        } `
+      : "";
+
+    return `${condition} ${conceptType} ${system}`;
+  };
+
+  const renderValueSetRows = () => {
+    return paginatedValueSets.map((vs, index) => (
+      <tr
+        key={index}
+        className={classNames(
+          styles.valueSetTable__tableBody_row,
+          vs?.valueSetId == activeValueSet?.valueSetId
+            ? styles.activeValueSet
+            : "",
+        )}
+        onClick={() => setActiveValueSet(vs)}
+      >
+        <td>
+          {/* TODO: build out search to include match on code terms within a value set? */}
+          <div className={styles.valueSetTable__tableBody_row_details}>
+            <Highlighter
+              className={styles.valueSetTable__tableBody_row_valueSetName}
+              highlightClassName="searchHighlight"
+              searchWords={[textSearch]}
+              autoEscape={true}
+              textToHighlight={vs.valueSetName}
+            />
+            <Highlighter
+              className={styles.valueSetTable__tableBody_row_valueSetDetails}
+              highlightClassName="searchHighlight"
+              searchWords={[textSearch]}
+              autoEscape={true}
+              textToHighlight={formatValueSetDetails(vs)}
+            />
+            {/* TODO: render based on the user_created column once that is added */}
+            {false && (
+              <Highlighter
+                className={styles.valueSetTable__tableBody_row_customValueSet}
+                highlightClassName="searchHighlight"
+                searchWords={[textSearch]}
+                autoEscape={true}
+                textToHighlight={`Created by ${vs.author}`}
+              />
+            )}
+          </div>
+          <div>
+            <Icon.NavigateNext
+              aria-label="Right chevron indicating additional content"
+              size={4}
+            />
+          </div>
+        </td>
+      </tr>
+    ));
+  };
+
+  return (
+    <WithAuth>
+      <div className={classNames("main-container__wide", styles.mainContainer)}>
+        <div className={styles.header}>
+          {mode === "manage" ? (
+            <Backlink onClick={() => {}} label={"Back to Query library"} />
+          ) : (
+            <Backlink onClick={goBack} label={"Back to My queries"} />
+          )}
+          <h1 className={styles.header__title}>Manage codes</h1>
+          <div className={styles.header__subtitle}>
+            Click on the checkbox to delete the value set or code
```

Comment:
@mikang this is the copy from Figma but wondering if we want something different for this view-only/manage mode?